src/github.com/getlantern/flashlight/proxiedsites/proxiedsites.go

```diff
@@ -52,6 +56,24 @@ func Configure(cfg *proxiedsites.Config, proxyAddr string) {
 	startMutex.Unlock()
 }
 
+func updateDetour(delta *proxiedsites.Delta) {
+	// TODO: subscribe changes of geolookup and set country accordingly
+	// safe to hardcode here as IR has all detection rules
+	detour.SetCountry("IR")
```

Comment:
Two detector actually. One for Iran and one for all other countries. Hardcode IR will add small performance penalty for other countries.